cdap-ui/app/cdap/components/DataPrepConnections/BigQueryConnection/BigQueryConnection.scss

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright © 2018 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+$asterisk-color: #d50000;
+$success-color: #00c853;
```

Comment:
Seems like these two colors are used in all the `.scss` files for DataPrep connections. Maybe we can move them to `variables.scss`?

Reply:
I will refactor this later. The entire DataprepConnections modals styling can be consolidated.